src/Parser/Core/Modules/Items/EnchantChecker.js

```diff
@@ -11,37 +11,25 @@ import SUGGESTION_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
 class EnchantChecker extends Analyzer {
 
   static ENCHANTABLE_SLOTS = {
-    1: 'Neck',
     10: 'Ring',
     11: 'Ring',
-    14: 'Cloak',
+    15: 'MainHand',
+    16: 'OffHand',
```

Comment:
have you checked what happens for specs that don't have an offhand (e.g. those that use a 2h) or only sometimes have an offhand (like mistweavers, who can either use an int 2h or 1h + oh)? in legion there was never a situation where you could be missing an enchantable slot